tools/speech_data_simulator/conf/data_simulator.yaml

```diff
@@ -52,6 +52,23 @@ data_simulator:
     background_manifest: null # Path to background noise manifest file
     num_noise_files: 10 # Number of randomly chosen noise source files to be potentially included in one session
     snr: 60 # SNR for background noise (using average speaker power)
+    snr_min: null  # Min random SNR for background noise (using average speaker power)
+    snr_max: null  # Max random SNR for background noise (using average speaker power)
+
+  # segment and session augmentations, see datails in https://github.com/NVIDIA/NeMo/blob/main/tutorials/asr/Online_Noise_Augmentation.ipynb
+  add_seg_aug: False  # set True to enable augmentation on each speech segment
+  segment_augmentor:
+    gain:
+      prob: 0.5
+      min_gain_dbfs: -10.0
+      max_gain_dbfs: 10.0
+
+  add_sess_aug: False # set True to enable audio augmentation on the whole session
+  session_augmentor:
```

Comment:
Please add some descriptions here for each item, to let the users know what it really does